pipeline-scripts/buildlib.groovy

```diff
@@ -130,7 +130,7 @@ def doozer(cmd, opts=[:]){
             return commonlib.shell(
                     returnStdout: opts.capture ?: false,
                     alwaysArchive: opts.capture ?: false,
-                    script: "doozer --assembly stream ${cleanWhitespace(cmd)}")
+                    script: "doozer --assembly=${params.ASSEMBLY ?: 'stream'} ${cleanWhitespace(cmd)}")
```

Comment:
The previous recommendations to remove the empty check do not apply here. A check here ensures that jobs that do not (and will not because it doesn't make sense in their context) have an `ASSEMBLY` parameter can still call doozer.